### DISCUSSION TITLE
Bug: Inconsistent order of commits (branch vs PR)

### DISCUSSION THREAD

**Discussion by MartinDahlborgCHR** [community/community#63519] [Pull Requests]
### Select Topic Area

Bug

### Body

**Description:**
As a developer jumping between GitHub pages, I want to have consistent and correct information at hand.

**Suggested solution:**
Please make both (all) commit-lists "Newest on top"

**Steps to reproduce:**
The code-branch-commits are displayed "Newest on top"
https://github.com/pytorch/xla/pull/5428/commits

However, when looking at the commits for a PR, they are displayed in the reverse order.
https://github.com/pytorch/xla/pull/5428/commits

**Problem statement:**
This adds undo strain on developers who have to remember that one page does this and another does the opposite. This bug introduces risk for misunderstandings, and therefore slower progress.

Thank you in advance,

**Comment by GUdokermensz** [comment]
ในวันที่ ส. 12 ส.ค. 2023 02:39 น. Martin Dahlborg CHR <
***@***.***> เขียนว่า:

**Comment by GUdokermensz** [comment]
***@***.***

ในวันที่ ส. 12 ส.ค. 2023 02:39 น. Martin Dahlborg CHR <
***@***.***> เขียนว่า:
